### PR TITLE
Preview device design fixes

### PR DESCRIPTION
**Pull request by punyakrit**
Iphone device preview #25

### CODE REVIEW

**Comment by punyakrit** [comment]
Screenshot 2024-08-14 at 2 39 39 PM

File: client/src/components/IframePreview.jsx [hunk 1]
```diff
@@ -30,18 +30,20 @@ const IframePreview = forwardRef(({ slug, isLoading, device = 'iPhone X' }, ref)
     <iframe
       ref={iframeRef}
       src={`${import.meta.env.VITE_BACKEND_URL}/site/${slug}/`}
-      className="w-full h-full border-0"
+      className={`w-full h-full border-0  ${device === "iPhone X" && "pt-8 bg-black"}`}
```
Comment (ptmaroct) [inline client/src/components/IframePreview.jsx:33]: can add a hasNotch or needsSafeAreaAdjustment array which includes "iPhone X" and add a check here if current device is part of that array then add the padding top bg etc

File: client/src/components/IframePreview.jsx [hunk 2]
```diff
@@ -30,18 +30,20 @@ const IframePreview = forwardRef(({ slug, isLoading, device = 'iPhone X' }, ref)
...
     </div>
   );
 
+
+
   return (
     <div className="w-full h-full flex items-center justify-center">
       <div className="scale-[0.7]">
-        <DeviceFrameset device={device}>
+        <DeviceFrameset device={device}  >
```
Comment (ptmaroct) [inline client/src/components/IframePreview.jsx:46]: extra space due to running formatter or added by mistake?

**Review by ptmaroct — CHANGES_REQUESTED** [review]

File: client/src/components/IframePreview.jsx (hunk 2 above)
Comment (punyakrit) [reply client/src/components/IframePreview.jsx:46]: this was by mistake

**Review by ptmaroct — APPROVED** [review]

**Comment by ptmaroct** [comment]
Merged, thanks for the fix ❤️

**Pull request merged**